ansible/prepare_host_env.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+if [ "$EUID" -ne "0" ]
+	then echo "Please run this setup as root"
+	return
+fi
+
+
+### Install required packages
+echo "Installing required packages..."
+
+# git
+apt install git -y
+
+# golang
+#set -e
+
+VERSION="1.9"
+PACKAGE="go$VERSION.linux-amd64.tar.gz"
+
+echo "Downloading $PACKAGE ..."
+wget https://storage.googleapis.com/golang/$PACKAGE -O /tmp/go.tar.gz
+if [ $? -ne 0 ]; then
+    	echo "Download failed! Exiting."
+        exit 1
+fi
+echo "Extracting go.tar.gz"
+tar -C "$HOME" -xzf /tmp/go.tar.gz
+
+# Be sure to uninstall old go
+rm -rf "$HOME/.go"
+
+# Save Go to .go
+mv "$HOME/go" "$HOME/.go"
+touch "$HOME/.bashrc"
+{
+	echo 'export GOROOT=$HOME/.go'
+	echo 'export PATH=$PATH:$GOROOT/bin'
+	echo 'export GOPATH=$HOME/go'
```

Comment:
@Dual2Core remove, this is default value for golang >= 1.8, `glide` might need to be updated. See PR: https://github.com/Masterminds/glide/issues/771